orbit-2-core/src/main/java/com/babylon/orbit2/RealContainer.kt

```diff
@@ -33,8 +35,8 @@ import kotlinx.coroutines.sync.withLock
 @Suppress("EXPERIMENTAL_API_USAGE")
 open class RealContainer<STATE : Any, SIDE_EFFECT : Any>(
     initialState: STATE,
-    settings: Container.Settings,
-    private val orbitDispatcher: CoroutineDispatcher = DEFAULT_DISPATCHER,
+    private val settings: Container.Settings,
+    orbitDispatcher: CoroutineDispatcher = DEFAULT_DISPATCHER,
```

Comment:
What happened here with the visibility?

Reply:
`orbitDispatcher` didn't need to be `private val` as its only used at construction time

Reply:
Looked at it the wrong way hehe I forgot you removed `val` as well